lstmcpipe/config_MC_prod.yml

```diff
@@ -18,33 +18,23 @@ workflow_kind: lstchain  # Not expected to be changed by the user
 base_path: /fefs/aswg/workspace/MC_data_simlink  # Default user path
   # lstanalyzer user : /fefs/aswg/data/mc
   #
-# Only for dl1 reprocessing
-#dl1_reference_id: #old id including versions (full directory name)
 prod_type: prod5
-  # Choose between {prod5, prod3}
 obs_date: 20200629_prod5_trans_80
-  # prod5 : `20200629_prod5` or `20200629_prod5_trans_80`
```

Comment:
Sorry for being picky here. 
I would suggest to keep some (a less as possible) comments indicating the possible values of certain fields  (users usually are not aware of how many MC simtel prods are). 

Maybe `prod_type` and `obs_date` ? What do you think ? Not sure if anybody would like to run old prods, indeed...

Reply:
But why would anyone run anything else than these values?
Its not like we have 200 simulations currently

Reply:
But maybe obs date is not self-explanatory indeed

Reply:
If they dont know the productions they should stick to the defaults though I think

Reply:
Fair enough ! you convinced me :)